content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst

```diff
@@ -0,0 +1,318 @@
+
+Comparison
+----------
+
+The options in the :guilabel:`Comparison` section are used to compare production data from two
+different time periods. There are two comparison options to choose from: :guilabel:`End Date:
+Previous Period` and :guilabel:`End Date: Previous Year`.
+
+To view data using one of the two comparisons, begin by selecting a time period in the
+:guilabel:`End Date` field of the :guilabel:`Search...` bar. Then, make sure the
+:icon:`fa-pie-chart` :guilabel:`(pie chart)` graph type is selected.
+
+Finally, select either :guilabel:`End Date: Previous Period` or :guilabel:`End Date: Previous Year`
+in the :guilabel:`Comparison` section of the :guilabel:`Search...` bar drop-down menu.
+
+With one of the :guilabel:`Comparison` options enabled, the pie chart appears as a large circle with
+a smaller circle inside of it. The larger circle represents the selected time period, while the
+smaller circle represents the same unit of time (month, quarter, year), one period or year previous.
+
+.. example::
+   :guilabel:`Q3` is selected in the :guilabel:`End Date` filter section of the
+   :guilabel:`Search...` bar. In the :guilabel:`Comparison` section, :guilabel:`End Date: Previous
+   Year` is selected.
+
+   The current year is 2024, so the larger circle shows data for the third quarter (Q3) of 2024. The
+   smaller circle shows data for the third quarter of 2023, which is the same unit of time, but one
+   *year* previous.
+
+   If :guilabel:`End Date: Previous Period` is selected instead, the smaller circle shows data for
+   second quarter of 2024, which is the same unit of time, but one *period* previous.
+
+.. important::
+   The :guilabel:`Comparison` section only appears in the :guilabel:`Search...` bar drop-down menu if
+   one or more time periods is selected in the :guilabel:`Filters --> End Date` section. This is
+   because a time period must be selected, or else there would be nothing to make a comparison to.
+
+   Additionally, the comparison view is meant to be used with the :icon:`fa-pie-chart`
+   :guilabel:`(pie chart)` graph type selected. Selecting a comparison period with a different graph
+   type enabled does not change the way the data appears on the page.
+
+.. image:: production_analysis/comparison.png
+   :align: center
+   :alt: The comparison view of the Production Analysis report.
+
+Measures
+========
+
+*Measures* are the datasets that can be selected in the production analysis report. Each dataset
+represents a specific statistic about |MOs| in the database. Choose a measure by clicking the
+:guilabel:`Measures` :icon:`fa-caret-down` button, and selecting one of the following options from
+the drop-down menu:
+
+- :guilabel:`Average Component Cost/Unit`: the average cost of the components required to produce
+  one unit of the product.
+- :guilabel:`Average Employee Cost/Unit`: the average cost paid to employees to produce one unit of
+  the product.
```

Comment:
```suggestion
- :guilabel:`Average Employee Cost/Unit`: the manual labor cost to produce one unit, calculated using the employee's hourly rate and the time spent on operations
```

Reply:
This is not necessarily true. There are other ways that Odoo computes employee cost, depending on certain configuration options. I think it's a bit too much to explain on this one bullet point though.